app/testonly/controllers/FeatureSwitchController.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package testonly.controllers
+
+import config.AppConfig
+import config.featureSwitch.FeatureSwitchModel
+import play.api.libs.json.Json
+import play.api.mvc.{Action, AnyContent, ControllerComponents, Result}
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
+
+import javax.inject.Inject
+
+class FeatureSwitchController @Inject()(appConfig: AppConfig, cc: ControllerComponents) extends BackendController(cc) {
```

Comment:
A controller would only be required if we are expecting another service to send a request to financial-transactions to change the feature switch values. As of now we didn't decide on that. If we do it later down the line then let's add that functionality at the time. As of now this can be removed